website/docs/reference/commands/clone.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "About dbt clone command"
+sidebar_label: "clone"
+id: "clone"
+---
+
+The `dbt clone` command clones selected nodes from the specified state to the target schema(s). This command makes use of the `clone` materialization:
+- If using a data warehouses that support zero-copy cloning of tables, selected models that are tables will be created as clones
+- For views and warehouses that do not support zero-copy cloning, selected models will be created as simple `select * from ...` pointer views
+
+Note: 
+- The state to clone from is based on the location of nodes in the manifest provided to `--state`.
+- By default, `dbt clone` will not recreate pre-existing relations in the current target. To override this, use the `--full-refresh` flag. 
+- You may want to specify a higher number of [threads](website/docs/docs/running-a-dbt-project/using-threads.md) to decrease execution time since individual clone statements are independent of one another.
```

Comment:
```suggestion
- You may want to specify a higher number of [threads](/docs/running-a-dbt-project/using-threads) to decrease execution time since individual clone statements are independent of one another.
```

Reply:
> individual clone statements are independent of one another.

They _ought_ to be, although this isn't totally true in practice (yet). We still need to actually make this happen:
- https://github.com/dbt-labs/dbt-core/issues/7914

But definitely worth calling out that, with parallelism in the mix, we expect this to be _significantly faster_ than `create schema clone` in a `run-operation`. (For dbt Labs' internal analytics project, I found it to be 60-80% faster.)